src/custom/state/network/updater.ts

```diff
@@ -0,0 +1,11 @@
+import { useActiveWeb3React } from 'hooks'
+import { switchParamsByNetwork } from '../../utils/xdai/hack'
+
+export default function Updater(): null {
```

Comment:
Why does it return `null` instead of `void`?

Reply:
Because it's a react component

Reply:
exactly, see how also the other Updaters return null

![image](https://user-images.githubusercontent.com/2352112/105682542-e0b53280-5ef2-11eb-800e-e07331d61899.png)

Reply:
ohhhhh ok